subgraph/subgraph.template.yaml

```diff
@@ -22,7 +23,37 @@ dataSources:
         - name: Unlock
           file: ./abis/UnlockV11.json
         - name: PublicLock
-          file: ./abis/PublicLock.json
+          file: ./abis/PublicLockV13.json
+        - name: PublicLockV11
+          file: ./abis/PublicLockV11.json
+        - name: PublicLockV7
+          file: ./abis/PublicLockV7.json
+      eventHandlers:
+        - event: NewLock(indexed address,indexed address)
+          handler: handleNewLock
+        - event: LockUpgraded(address,uint16)
+          handler: handleLockUpgraded
+        - event: GNPChanged(uint256,uint256,address,uint256,address)
+          handler: handleGNPChanged
+      file: ./src/unlock.ts
+  - kind: ethereum
+    name: Unlock2
```

Comment:
Same comment about the array per above?

Reply:
This now generates for 1 to many Unlock versions from a deployments array.